app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -405,6 +342,21 @@ class VaultRepositoryImpl(
         }
     }
 
+    override suspend fun syncFido2Credentials(): SyncFido2CredentialsResult {
+        val userId = activeUserId
+            ?: return SyncFido2CredentialsResult.Error(throwable = null)
+        syncJob = ioScope
+            .async { syncInternal(userId) }
+            .also {
+                return try {
+                    it.await()
+                } catch (e: CancellationException) {
+                    currentCoroutineContext().ensureActive()
```

Comment:
What does this do?

Reply:
It's used to more granularly handle cancellation. `ensureActive()` will throw if `ioScope` coroutine is cancelled. That allows us to handle cancellation of the `Deferred<T>` returned by `await()` and let `ioScope` callers handle its cancellation. 

We don't need it in this case. I forgot to remove it. Regardless of where the cancellation comes from, we are going to return an Error.